### PR TITLE
Changed to execFile and make cmd as list/array

### PR DESCRIPTION
**Pull request by YanDevDe**
from issue https://github.com/Alex-D/check-disk-space/issues/6

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -54,15 +54,15 @@ function mapOutput(stdout, filter, mapping, coefficient = 1) {
 /**
  * Run the command and do common things between win32 and unix
  *
- * @param {String} cmd - The command to execute
+ * @param {String} cmd - The command to execute (must be in list/array)
```
Comment (sergiz) [inline index.js:57]: You should do
`@param {String[]} cmd`
for a proper JSDoc.

Comment (YanDevDe) [reply index.js]: Thanks, I don't use JSDoc!

I also applied the changes you requested.

**Comment by Slapbox** [comment]
@Alex-D

Or @BrunnerLivio if you have merge permissions.

I doubt everyone using this package is perusing the issues page, so it would be great to have this fixed in the mainline version rather than having to rely on a fork.

**Review by sergiz — APPROVED** [review]

**Comment by Alex-D** [comment]
Thank you for your Pull Request!

I have taken this in account in my rewrite of the package.

Now, it is full TypeScript, with builds for CommonJS and ESM, and generated .d.ts.

**Comment by Slapbox** [comment]
image

Thanks to @Alex-D and @YanDevDe!

**Pull request closed without merging**